### PR TITLE
BCN 04/19 - Yvonne Moser

### PR DESCRIPTION
**Pull request by mucklaa**

### CODE REVIEW

**Comment by mucklaa** [comment]
failure in Jasmine: 

Reveal the mistery - revealMistery Return <FIRST NAME> <LAST NAME> killed Mr.Boddy using the <WEAPON> in the <PLACE>!!!!

Expected 'Victor Plum killed Mr. Boddy using the poison in the [object Object]!!!!' to equal 'Victor Plum killed Mr.Boddy using the poison in the Billiard Room!!!!'

File: src/clue.js [hunk 1]
```diff
@@ -1,94 +1,121 @@
...
+    first_name:   "Jack",
+    last_name:    "Mustard",
+    color:        "yellow",
+    description:  "He is a former football player who tries to get by on his former glory",
+    age:          62,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation:   "Retired Football player"
+  }
+
+charactersArray.push(mrGreen);
```
Comment (tawebbcn) [inline src/clue.js:65]: Very interesting way to do it :)

File: src/clue.js [hunk 2]
```diff
@@ -1,94 +1,121 @@
...
+
+
+function randomSelector(array){
+  var randomElement = array[Math.floor(Math.random() * array.length)];
+  return randomElement;
+};
+
+// check function: console.log(randomSelector(weaponsArray));
+
+function pickMistery(){
```
Comment (tawebbcn) [inline src/clue.js:101]: From the review I think you can get how to solve this part!

**Review by tawebbcn — COMMENTED** [review]
Looks good. From the things we've reviewed in the morning try to refactor the code and finish it, you can do it!

**Pull request closed without merging**